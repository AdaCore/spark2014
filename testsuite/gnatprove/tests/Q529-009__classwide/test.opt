ALL XFAIL crash in Why3 and flow error
