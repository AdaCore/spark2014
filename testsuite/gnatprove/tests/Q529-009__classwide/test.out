p_heir1.adb:16:41: medium: overflow check might fail (e.g. when This = (X1 => 2147483647))
p_heir1.adb:17:41: medium: overflow check might fail
p_heir2.adb:16:41: medium: overflow check might fail (e.g. when This = (X2 => 2147483646))
p_heir2.adb:17:41: medium: overflow check might fail
p_parent.adb:16:41: medium: overflow check might fail (e.g. when This = (X0 => 2147483647))
p_parent.ads:11:22: warning: unused initial value of "This"
p_u_parent.ads:11:24: "memory accessed through objects of access type" is an input of overriding subprogram, but it is not an input of overridden subprogram "Get_X0" at p_i_parent.ads:9 (SPARK RM 6.1.6)
p_u_parent.ads:13:25: "memory accessed through objects of access type" is an in_out of overriding subprogram, but it is not an input of overridden subprogram "Set_X0" at p_i_parent.ads:11 (SPARK RM 6.1.6)
p_u_parent.ads:13:25: "memory accessed through objects of access type" is an in_out of overriding subprogram, but it is not an output of overridden subprogram "Set_X0" at p_i_parent.ads:11 (SPARK RM 6.1.6)
p_u_parent.ads:16:25: "memory accessed through objects of access type" is an in_out of overriding subprogram, but it is not an input of overridden subprogram "Compute" at p_i_parent.ads:14 (SPARK RM 6.1.6)
p_u_parent.ads:16:25: "memory accessed through objects of access type" is an in_out of overriding subprogram, but it is not an output of overridden subprogram "Compute" at p_i_parent.ads:14 (SPARK RM 6.1.6)
gnatprove: error during flow analysis and proof
